.gitignore

```diff
@@ -28,3 +28,4 @@ src-api/build/
 vendor
 jekyll/_config_override.yml
 markdownlint-results.json
+.tool-verisons
```

Comment:
```suggestion
.tool-versions
```

Reply:
So that's why it wasn't picking it up... 😂